lib/resources/r4/communication.yaml

```diff
@@ -9,102 +9,86 @@ fields:
   description: The type of the FHIR resource.
   url: https://hl7.org/fhir/r4/Communication-definitions.html#Communication
   note: resourceType must be Communication.
+  action: create
   example: |
     {
       "resourceType": "Communication"
     }
-  action:
-    - create
 
-- name: inResponseTo
-  required: 'no'
-  cardinality: 0..1
-  type: Reference(Communication)
-  description: The Communication this Communication is in response to.
-  url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.inResponseTo
-  note: 
-  example: |
-    {
-      "inResponseTo": {
-        "reference": "Communication/23456_45678",
-        "type": "Communicaiton"
-      }
-    }
-  action:
-    - create
 
```

Comment:
nit: extra line

Reply:
I moved all my 'action: -create' to be one line(https://github.com/cerner/fhir.cerner.com/pull/435/commits/6bfcb726af06ef17098ae7bdac74a5ff2852ac2b#)...I think the extra blank line you're talking about got removed in rebase, you can see it's not there now: https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-f08ae1e4c063b9a36a22adaa43fecf21L18